content/applications/sales/crm/track_leads/lead_distribution_report.rst

```diff
@@ -0,0 +1,302 @@
+
+This rule includes any leads with the above terms in the attached notes to the report.
+
+.. _tags:
+
+4. Tags
+~~~~~~~
+
+Click the :icon:`fa-plus` :guilabel:`(New Rule)` icon next to line for `Notes contains fsm`.
+
+Next, set the first field to `Tags`. Select :guilabel:`contains` from the drop-down menu in the
+second field. Then, type `20` in the third field.
+
+This rule adds any leads that are tagged as a size of 5-20 or 20-50 to the report.
+
+.. _email:
+
+5. Email
+~~~~~~~~
+
+Click :guilabel:`New rule` at the bottom of the pop-up menu to add a new rule. This new rule is
+created outside the :guilabel:`any of:` group. Set the first field to :guilabel:`Email`. Select
+:guilabel:`does not contain` from the drop-down menu in the next field. Next, type `hotmail` into
+the rightmost field.
+
+Repeat the previous steps to add rules for the following contents:
+
+#. aol.com
+#. icloud.com
+#. yahoo
+
+.. image:: lead_distribution_report/add-email.png
+  :align: center
+  :alt: Add rules to filter out the above four email domains.
+
+This rule filters out any undesired email addresses from the report.
+
+.. _salesperson:
+
+6. Salesperson
+~~~~~~~~~~~~~~
+
+Click :guilabel:`New rule` to add a new rule. Click the first field, then locate and select
+:guilabel:`Salesperson`. Click the :icon:`fa-chevron-right` :guilabel:`(right chevron)` button to
+the right. A new list of salespeople attributes appears. Select :guilabel:`Active` from the list.
+In the second field, select :guilabel:`is` from the drop-down menu. In the third drop-down menu,
+select :guilabel:`set`.
+
+.. image:: lead_distribution_report/add-salesperson.png
+  :align: center
+  :alt: Add a salesperson's activity status to ensure an active salesperson is selected.
+
+This rule ensures only leads with an active salesperson are added to the report.
+
+Save the filter settings
+========================
+
+Now that all the rules for the filter are set, click the purple :guilabel:`Confirm` button at the
+bottom of the pop-up menu to save the custom filter and close the pop-up menu.
+
+.. image:: lead_distribution_report/confirm-filter.png
+   :align: center
+   :alt: Click the confirm button to save the filter.
+
+
+The :guilabel:`Pipeline Analysis` dashboard is now displayed again with each filter rule in the
+search bar. Click the :icon:`fa-caret-down` :guilabel:`(down caret)` icon, to the right of the
+search bar, to reveal a drop-down menu of search and filter options.
+
+Scroll to the bottom and click the :guilabel:`Save current search` button. Next, type a name for the
+filter in the text box below. Check the :guilabel:`Shared` checkbox to have the filter shared with
+any user with access to the pipeline. Finally, click the purple :guilabel:`Save` button below to
+save the filter.
+
+The filter will now appear with the name it was given under the :guilabel:`Favorites` section of the
+drop-down menu and can be re-applied by clicking on it.
+
+View lead distribution reports
```

Comment:
I might have missed it but after adding the custom Filters it's necessary to do Group Bys afterwards.